compiler/rustc_query_impl/Cargo.toml

```diff
@@ -25,5 +24,8 @@ rustc_span = { path = "../rustc_span" }
 thin-vec = "0.2.12"
 tracing = "0.1"
 
+# Not used directly, but included to enable the unstable_offset_of feature
```

Comment:
is the comment still accurate? this crate had the dep before

Reply:
Grep for `memoffset`, there is no results.

Before it was included to enable the `unstable_const` feature, which [<enabled usage of the `pointer::offset_from` function](https://github.com/Gilnaa/memoffset/blob/6afc606b09d293ae452c29f82c72cb0362bd6190/src/offset_of.rs#L48-L69), but that function is not used if `unstable_offset_of` is enabled as that directly uses `offset_of!()`.

The `offset_of_tuple` macro still doesn't use `offset_of` so `unstable_const` does *something* to the public API of `memoffset` still, but that macro is not used I think neither by rustc nor by any of its deps, at least that's my impression.

In other words, enabling `unstable_const` makes no sense at this point.